python.html.markdown

```diff
@@ -67,23 +72,26 @@ allow you to write Python 3 code that will run on Python 2, so check out the Pyt
 # Enforce precedence with parentheses
 (1 + 3) * 2  # => 8
 
-# Boolean Operators
+# Boolean Operators, which are type bool
```

Comment:
Boolean operators are not of type bool. The literals are, but the operators are not.